### PR TITLE
Add testing

### PR DESCRIPTION
**Pull request by mwestphal**
Add F3D package testing

### CODE REVIEW

File: projects/rkcommon.cmake [hunk 1]
```diff
@@ -0,0 +1,11 @@
+superbuild_add_project(rkcommon
+  LICENSE_FILES
+    LICENSE.txt
+  DEPENDS cxx11 tbb
+  CMAKE_ARGS
+     -DBUILD_TESTING:BOOL=OFF
+     -DCMAKE_INSTALL_LIBDIR:STRING=lib
+     -DCMAKE_INSTALL_NAME_DIR:PATH=<INSTALL_DIR>/lib
+     -DINSTALL_DEPS:BOOL=OFF
+     -DCMAKE_INSTALL_RPATH=<INSTALL_DIR>/lib
```
Comment (mwestphal) [inline projects/rkcommon.cmake:10]: should be added to common superbuild in the future

Comment (mwestphal) [reply projects/rkcommon.cmake]: https://gitlab.kitware.com/paraview/common-superbuild/-/merge_requests/544

File: .gitattributes [hunk 2]
```diff
@@ -1 +1,5 @@
 *.patch     whitespace=-blank-at-eol,-space-before-tab -text -crlf
+
+testing/data/**/* filter=lfs diff=lfs merge=lfs -text
+testing/baselines/**/* filter=lfs diff=lfs merge=lfs -text
+testing/data/DATA_LICENSES.md !filter !diff !merge text
```
Comment (mwestphal) [inline .gitattributes:5]: this file is missing

**Review by Meakk — APPROVED** [review]
Looks good :+1:

**Pull request merged**